package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "health-echo",
+  "version": "1.0.0",
+  "description": "This README would normally document whatever steps are necessary to get the application up and running.",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "rspec && npm run-script test --prefix react",
+    "install": "npm install --prefix react",
+    "start": "heroku local -f Procfile.dev",
+    "build": "npm install --prefix react && npm run-script build --prefix react",
+    "deploy": "cp -a react/build/. public/",
+    "heroku-postbuild": "npm run-script build && npm run-script deploy"
+  },
+  "engines": {
+    "npm": "6.14.4",
+    "node": "14.0.0"
+  },
```

Comment:
No need to include the engines section, but if so you probably want to go with the lowest possible versions that can still work on this code, instead of forcing users to upgrade to these versions when older versions will work fine.

Reply:
Apparently, they were necessary for uploading the app to the Heroku.